README.md

```diff
@@ -1,51 +1,127 @@
+[![Compile to different clouds](./logo/compile-diff-clouds.gif)](https://youtu.be/gr4ewOJGkfo)
+
+## Local simuation
+Wing provides a built-in local simulator, and an observability & debugging [console] for easy testing and fast iteration loops.
+
+[![Compile to different clouds](./logo/console.gif)](https://youtu.be/Gqn1hYPEwqg)
+
+## Cloud abstraction
+While most programming languages think about computers as machines, in Wing, **_the cloud is the computer_**.
+Similar to how traditional languages abstract the computer, wing abstarcts the cloud with a [standard library] of the basic cloud services. The Wing compiler uses a dependency injection mechanism to convert these abstract services into the configured ones for each target cloud. It is possible to create and use external libraries too, including import of any Terraform resource.
 
 ```js
+// This line imports the standard library.
 bring cloud;
 
+// We are instantiating an abstract bucket here.
+// At compile time, it will be substituted by the configured implementation for the target cloud.
 let bucket = new cloud.Bucket();
+```
 
-new cloud.Function(inflight (event: str): str => {
-  bucket.put("greeting.txt", "hello, world!");
+Since the Wing compiler sees the entire application, including its infrastrucrure, it can also generate least privilege IAM policies based on intent, and other cloud mechanics.
+
+```js
+bring cloud;
+
+// This code runs at compile time to provision the bucket.
+let bucket = new cloud.Bucket();
+
+// This code runs at compile time to provision the function.
+new cloud.Function(inflight () => {
+  // This code runs later, on other machines, and interacts with the provisioned bucket.
+  // The compiler knows to generate the network topology and IAM policies needed for the function to be able to put
+  // a file in the bucket. It also knows to capture the bucket at deployment time and pass the info that is needed
+  // for the function to interact with it at runtime.
+  bucket.put("hello.txt", "World!");
 });
 ```
 
+## Wing's creator
+While there are many developers [contributing](#contributors) to the project, it started with the vision of [Elad Ben-Israel], who is also the creator of the [AWS CDK].
+
+[cloud-oriented programming language]: https://docs.winglang.io/#what-is-a-cloud-oriented-language
+[distributed computing primitives]: https://docs.winglang.io/concepts/inflights
+[resource system]: https://docs.winglang.io/concepts/resources
+[other optimizations for cloud programming]: #main-features-that-make-wing-tailored-for-cloud-development
+[compiler plugins]: https://docs.winglang.io/reference/compiler-plugins
+[Elad Ben-Israel]: https://github.com/eladb
+[AWS CDK]: https://github.com/aws/aws-cdk
+[console]: https://docs.winglang.io/getting-started/console
+[Standard library]: https://docs.winglang.io/reference/wingsdk-spec
+
+## Why a language? (and not an SDK or platform)
```

Comment:
```suggestion
## Why a language? (and not a library or platform)
```

When people ask "the question", I feel like I usually hear "library" used instead of SDK. Not sure I necessarily agree, but this question should match what people ask.